spec/controllers/course/assessment/assessments_controller_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe Course::Assessment::AssessmentsController do
+  let!(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let(:user) { create(:user) }
+    let!(:course) { create(:course, creator: user) }
+    let!(:immutable_assessment) do
+      create(:assessment, course: course).tap do |stub|
+        allow(stub).to receive(:destroy).and_return(false)
+      end
+    end
+
+    before { sign_in(user) }
+
+    describe '#update' do
+      subject { post :update, course_id: course, id: immutable_assessment,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.